code/src/components/TodoList.js

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from "react";
+import { useSelector, useDispatch } from "react-redux";
+import { CgClose } from "react-icons/cg";
+
+import todos from "../reducers/todos";
+
+import EmptyList from "../assets/emptyList.png";
+import SubHeader from "./SubHeader";
+
+import {
+  MainWrapper,
+  ItemList,
+  Paragraph,
+  Title,
+  CompleteTask,
+  SendButton,
+  ListWapper,
+  ParagraphWrapper,
+  EmptyListImage
+} from "../styled/StyledComponent";
+
+const TodoList = () => {
+  const items = useSelector((store) => store.todos.items);
+  const dispatch = useDispatch();
+  const [removeItems, setRemoveItems] = useState(false);
+
+  const dateConverter = `
+  ${new Date().toLocaleString("default", {
+    weekday: "long",
+    day: "2-digit",
+    month: "long"
+  })}`;
+
+  return (
+    <MainWrapper>
+      <Title>My Day</Title>
+      <Paragraph>{dateConverter}</Paragraph>
+      <SubHeader removeItems={removeItems} setRemoveItems={setRemoveItems} />
+
+      <ListWapper>
+        {items.length ? (
+          items.map((todo, index) => (
+            <ItemList key={todo.id}>
+              <ParagraphWrapper>
+                <Paragraph>{todo.taskDescription}</Paragraph>
+                <Paragraph
+                  style={{ fontSize: 10, marginTop: 0, marginBottom: 5 }}>
+                  {new Date(todo.timestamp).toLocaleString("default", {
+                    day: "2-digit",
+                    month: "long"
+                  })}
+                </Paragraph>
+              </ParagraphWrapper>
```

Comment:
You need to be ready for longer inputs, so you can wrap your line or add max-length to your input 
![image](https://user-images.githubusercontent.com/50926159/116785975-af717300-aa9c-11eb-9add-7319386be84d.png)